replicated-security/stride-ics-testnet-two-1/readme.md

```diff
@@ -0,0 +1,113 @@
+
+### How will the sovereign -> consumer chain transition work on the Replicated Security testnet?
+
+* Stride side: The Stride team is running their own testnet (chain-id: stride-ics-testnet-two-1). That testnet will perform a software upgrade and at the upgrade height (shortly after the spawn time) it will transition to the provider chain’s validator set.
+* Provider side: There will be a consumer-addition proposal for the stride chain. Shortly after the spawn time, we will receive the CCV state. This CCV state will be used to patch the original stride chain’s genesis file.
+
+### What do you need to do to participate in the launch on Wednesday?
+See the table below for a breakdown of steps you'll need to follow throughout the process. 
+
+## ⚠️  Complete STEP 1 (join Stride testnet with a full node) ASAP ⚠️
+Follow along with Stride's block explorer here: https://ics-explorer.stride.zone/stride 
+
+For step 1, you can try using Stride’s joining script here: https://github.com/Stride-Labs/mainnet/blob/ics-testnet/ics-testnet/join_ics_testnet.sh 
+
+Otherwise you may manually join stride-ics-testnet-two-1 using these notes:
+* Joining instructions: https://github.com/Stride-Labs/mainnet/tree/ics-testnet/ics-testnet
+* Genesis file: https://raw.githubusercontent.com/Stride-Labs/mainnet/ics-testnet/ics-testnet/genesis.json
+* Pre-transition stride binary commit: `a3eff2dc`
+* Stride’s GitHub repository: https://github.com/stride-Labs/stride
+* Building instructions for stride’s binary: `make install`
+* Go version: 1.19
+* Persistent Peers = `"d747545dbab1eb86caff7ec64fca3b7f2ace07fd@stride-direct.testnet-2.stridenet.co:26656"`
+
+* Chain ID: stride-ics-testnet-two-1
+* Post-upgrade stride binary commit (run with this binary after the upgrade): [`5a45a4cfaef54aa0e875eb4e618bfae882d278a2`](https://github.com/Stride-Labs/stride/commit/5a45a4cfaef54aa0e875eb4e618bfae882d278a2)
+  * You can also build with [this Docker image](https://hub.docker.com/layers/stridelabs/ics-testnet/stride/images/sha256-3268198b39fa9e3b6107f352f49d28c5c78939e1147370b166f848dbd112186e?context=repo)
+ 
+<details><summary>Detailed steps for joining Stride Testnet</summary>
+<br>
+ 
+ _Courtesy of Stakecito_
+
+```sh
+git clone https://github.com/Stride-Labs/stride.git
+cd stride
+git checkout a3eff2dc
+make install
+strided init stride-node --chain-id stride-ics-testnet-two-1
+
+# Grab the genesis file
+curl -L https://raw.githubusercontent.com/Stride-Labs/mainnet/ics-testnet/ics-testnet/genesis.json -o $HOME/.stride/config/genesis.json
+```
+
+* Start stride node, node should start catching up
+* Node will panic at UPGRADE_HEIGHT_TBD
+* Stop the node
+
+```sh
+cd $HOME/stride
+
+git checkout 5a45a4cfaef54aa0e875eb4e618bfae882d278a2
+
+make install
+```
+
+replace the binary
+
+```sh
+mkdir -p $HOME/.sovereign/config
```

Comment:
```suggestion
```

Reply:
We don't need to create this folder now, it's already there

Reply:
Actually I think we want to remove everything from line 59 to 65. Bosco's instructions below do a better job of describing the binary swap step